packages/client/src/utils/persistence/BrowserPersistence.ts

```diff
@@ -1,22 +1,24 @@
 import { get, set, createStore, UseStore } from 'idb-keyval'
 import { Persistence } from './Persistence'
 import { StreamID } from 'streamr-client-protocol'
+import { memoize } from 'lodash'
 
 export default class BrowserPersistence implements Persistence<string, string> {
-    private store: UseStore
-    private dbName: string
+    private getStore: (streamId: StreamID) => UseStore
 
-    constructor({ clientId, streamId }: { clientId: string, streamId: StreamID }) {
-        this.dbName = `streamr-client::${clientId}::${streamId}`
-        this.store = createStore(this.dbName, 'GroupKeys')
+    constructor({ clientId }: { clientId: string }) {
```

Comment:
Just a question, what is `clientId` in practice? Is it the public Ethereum address of the client?

Reply:
Yes, it is `authentication.getAddress()` (`GroupKeyStore:38`), not an optimal variable name